addOns/grpc/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 
 ## 0.1.0 - 2024-05-17
```

Comment:
This line should be removed, that will be done automatically during the release process.